questionnaire/questionnaireUISchema.js

```diff
@@ -1641,6 +1731,53 @@ module.exports = {
     },
     'p-rep-address': {
         options: {
+            properties: {
+                'q-rep-building-and-street': {
+                    options: {
+                        macroOptions: {
+                            classes: '',
+                            autocomplete: 'address-line1'
+                        }
+                    }
+                },
+                'q-rep-building-and-street-2': {
+                    options: {
+                        macroOptions: {
+                            classes: '',
+                            autocomplete: 'address-line2'
+                        }
+                    }
+                },
+                'q-rep-building-and-street-3': {
+                    options: {
+                        macroOptions: {
+                            classes: '',
+                            autocomplete: 'address-line3'
+                        }
+                    }
+                },
+                'q-rep-town-or-city': {
```

Comment:
Are these autocomplete levels correct ?
https://www.w3.org/TR/WCAG21/#input-purposes
address-level2 - The second administrative level, in addresses with two or more administrative levels; in the countries with two administrative levels, this would typically be the city, town, village, or other locality within which the relevant street address is found
address-level1 - The broadest administrative level in the address, i.e., the province within which the locality is found; for example, in the US, this would be the state; in Switzerland it would be the canton; in the UK, the post town
country - Country code
country-name - Country name

Reply:
It's fine, interesting that the guidelines incorrectly(?) mention in the UK, the post town. Wonder how it works when you save an address somewhere else?. And use the functionality on the cica app.